.travis.yml

```diff
@@ -76,6 +78,8 @@ script:
 
     - etc/bin/test-packages
 
+    - gulp admin-css admin-js
```

Comment:
Actually, we should remove ``watch`` task from ``default`` gulp task. Then we can run ``gulp`` to process everything and ``gulp watch`` to process everything and watch for changes. On travis we would simply run ``gulp``.